src/DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -2301,8 +2301,8 @@ function string RemapBingoEvent(string eventname)
         case "M04MeetSmuggler_Played":
         case "M08SmugglerConvos_Played":
             return "MeetSmuggler";
-        case "Ray_Dead":
-        case "Ray_Unconscious":
+        case "Ray_PlayerDead":
+        case "Ray_PlayerUnconscious":
```

Comment:
Like the other case, the old behaviour aligns with the conversation that it's associated with, so it should probably stay the same